types/react-native-maps/index.d.ts

```diff
@@ -0,0 +1,474 @@
+declare module "react-native-maps" {
```

Comment:
This file should start with headers that specify what this package is, its version, the maintainers, etc. You can see that CI is failing at the ‘parse header’ stage https://travis-ci.org/DefinitelyTyped/DefinitelyTyped/builds/533108975#L477

Copy it from the RN types and adjust the name and version number.

Reply:
Btw, there’s no need for `declare module "react-native-maps" { … }` when you have a `.d.ts` file that maps directly to the import of the module.

Reply:
Ok awesome. Thanks for the explanation!

I’ll take a look later today.